Algorithm.Python/QuandlImporterAlgorithm.py

```diff
@@ -0,0 +1,61 @@
+﻿# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import date
+
+from clr import AddReference
+AddReference("System")
+AddReference("QuantConnect.Algorithm")
+AddReference("QuantConnect.Indicators")
+AddReference("QuantConnect.Common")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Algorithm import *
+from QuantConnect.Indicators import *
+from QuantConnect.Data.Custom import *
+
+
+class QuandlImporterAlgorithm(QCAlgorithm):
+    '''QuantConnect University: Generic Quandl Data Importer
+    Using the underlying dynamic data class "Quandl" we take care of the data 
+    importing and definition for you. Simply point QuantConnect to the Quandl Short Code.
+     
+    The Quandl object has properties which match the spreadsheet headers.
+    If you have multiple quandl streams look at data.Symbol to distinguish them.'''
+    
+    __sma = None
+    __quandlCode = "YAHOO/INDEX_SPY"
```

Comment:
are these akin to static variables in c#? Ideally we wouldn't make examples using static values in algorithms as it may cause issues for people trying to run algorithms in parallel.